src/Workspaces/CSharp/Portable/LanguageServices/CSharpSemanticFactsService.cs

```diff
@@ -33,12 +33,12 @@ protected override IEnumerable<ISymbol> GetUsedSymbols(SemanticModel semanticMod
 
             // Some symbols in the enclosing block could cause conflicts even if they are not available at the location.
             // E.g. symbols inside if statements / try catch statements.
-            // Walk through the enclosing block to find them, but make sure to exclude local functions because
-            //     a) Visible local function symbols would be returned from LookupSymbols (e.g. location is inside a local function).
-            //     b) Local function symbols are only in scope inside the local function.
-            //        Relevant ones (e.g. method name) would be returned by a).
+            // Walk through the enclosing block to find them, but avoid exploring local functions because
+            //     a) Visible local function symbols would be returned from LookupSymbols
+            //        (e.g. location is inside a local function or the local function method name).
+            //     b) Local function symbols are not affected by outer variables and do not contribute to the outer scope.
```

Comment:
i think this could be worded slightly better.  when you say "local function symbols" it sounds like you're talking about the local function *itself*.  I think we should be clearer and say that "symbols declared **inside** a local function do not cause collisions with symbols declared **outside** teh local function.  And so, should not be considered here.

Note: GetUsedSymbols is not a great name either.  it should be something like GetCollidableSymbols since htat's what we're really asking for her.e  We should also doc teh GetCollidableSymbls method to state what the intent of it is.  That will help ensure that we have a clear path going forward in the future for how to fix it up if we get something wrong (or as the langauge changes).